### PR TITLE
fix: 2023 titles in 2024 pages

### PR DESCRIPTION
**Pull request by TheJoin95**

### CODE REVIEW

File: pages/cookie.tsx [hunk 1]
```diff
@@ -5,9 +5,7 @@ export async function getStaticProps({ locale }: { locale: any }) {
   return {
     props: {
       metas: {
-        title: 'Cookie, Open Source Day 2023 - Florence',
-        description:
-          'Open Source Day 2023 coming on the 24th of March 2023. Stay tuned on our social'
+        title: 'Cookie, Open Source Day 2043 - Florence',
```
Comment (Wabri) [inline pages/cookie.tsx]:
```suggestion
        title: 'Cookie, Open Source Day 2024 - Florence',
```
2043 you are a clairvoyant

File: pages/privacy-and-permissions.tsx [hunk 2]
```diff
@@ -6,9 +6,7 @@ export async function getStaticProps({ locale }: { locale: any }) {
     props: {
       metas: {
         title:
-          "Privacy and Permissions policies, Open Source Day 2023 - Florence",
-        description:
-          "Open Source Day 2023 coming on the 24th of March 2023. Stay tuned on our social",
+          "Privacy and Permissions policies, Open Source Day 2043 - Florence",
```
Comment (Wabri) [inline pages/privacy-and-permissions.tsx]:
```suggestion
          "Privacy and Permissions policies, Open Source Day 2024 - Florence",
```

**Review by Wabri — CHANGES_REQUESTED** [review]

File: pages/cookie.tsx (hunk 1 above)
Comment (TheJoin95) [reply pages/cookie.tsx]: Done, mr. president

**Review by Wabri — APPROVED** [review]
s'e'n'grande

**Pull request merged**